rasa/core/evaluation/marker_tracker_loader.py

```diff
@@ -0,0 +1,92 @@
+import random
+from rasa.shared.exceptions import RasaException
+from rasa.shared.core.trackers import DialogueStateTracker
+from typing import Any, Iterable, Iterator, List, Text, Optional
+from rasa.core.tracker_store import TrackerStore
+import rasa.shared.utils.io
+
+
+def strategy_all(count: int, keys: List[Text]) -> Iterable[Text]:
+    """Selects all keys from the set of keys."""
+    return keys
+
+
+def strategy_first_n(count: int, keys: List[Text]) -> Iterable[Text]:
+    """Takes the first N keys from the set of keys."""
+    return keys[:count]
+
+
+def strategy_sample(count: int, keys: List[Text]) -> Iterable[Text]:
+    """Takes a sample of N keys from the set of keys."""
+    return random.choices(keys, k=count)
+
+
+class MarkerTrackerLoader:
+    """Represents a wrapper over a `TrackerStore` with a configurable access pattern."""
+
+    _STRATEGY_MAP = {
+        "all": strategy_all,
+        "first_n": strategy_first_n,
+        "sample": strategy_sample,
+    }
+
+    def __init__(
+        self,
+        tracker_store: TrackerStore,
+        strategy: str,
+        count: int = None,
+        seed: Any = None,
+    ) -> None:
+        """Create a MarkerTrackerLoader.
```

Comment:
I can't find the notion page which says this, but I think this is the tone we prefer for docstrings.